### PR TITLE
Fixes #868: factor Header, SEO into PageBase

### LINKED ISSUE #868: Add the Header to /myfeeds
**What would you like to be added**:
Add the header to the my feeds page. 

image

**Why would you like this to be added**:
Currently, the page is completely isolated from the rest of the app. The header would at least will make for a better interface as it continues to be further integrated. It would also keep it consistent with the rest of the app, as all the other pages have the header.

### PR DESCRIPTION
**Pull request by Silvyre**
## Issue This PR Addresses

Fixes #868

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Removes references to our [`<Header>`](../tree/master/src/frontend/src/components/Header/Header.jsx) and [`<SEO>`](../tree/master/src/frontend/src/components/SEO.js) components from Gatsby Pages previously containing them, replacing them with a singular reference to those components within our `<PageBase>` wrapper component.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by humphd** [comment]
Tried on Zeit, works, but it suffers that bug with CSS making things huge:

Screen Shot 2020-04-02 at 7 14 36 PM

**Comment by agarcia-caicedo** [comment]
for some reason hovering over the home button (an only the home button) makes it shrink (in firefox)

**Comment by cindyledev** [comment]
That `src/frontend/src/pages/index.css` bug is affecting every page (myfeeds, search, the about page I'm working on in #896 ). You don't hit it as often in the Home page so most people might not notice but it's definitely present in every other page. I was hoping it would be fixed in #841 / #741 ...

I think this PR is good and does what it's suppose to do, it's just being affected by the css bug unfortunately

**Comment by cindyledev** [comment]
> for some reason hovering over the home button (an only the home button) makes it shrink (in firefox)

This happens in Chrome too... This is a neat find. I've started hovering over the Home button to test pages now
"Will my finger make everything small again?"
Yes it does... everytime... =_= 
I hate this bug so much....

**Comment by Silvyre** [comment]
Our CSS issues have been alleviated by https://github.com/Seneca-CDOT/telescope/pull/928; [check it out](https://telescope-ggyeojoab.now.sh/myfeeds/)!

File: src/frontend/src/pages/myfeeds.js [hunk 1]
```diff
@@ -51,6 +52,7 @@ export default function MyFeeds() {
 
   return (
     <PageBase>
+      <Header />
```
[the remark this answers is not in the document]
Comment (Silvyre) [reply src/frontend/src/pages/myfeeds.js:55]: You know what, we should... I'm going to refactor this PR to do so...

[the remark this answers is not in the document]
Comment (Silvyre) [reply src/frontend/src/pages/myfeeds.js]: Done in https://github.com/Seneca-CDOT/telescope/commit/8db0a24abe0d5f78249f4bb0da89177199457a04

**Comment by Silvyre** [comment]
> I see `<Header />` used in:
> 
> 1. Layout.js
> 2. Post.jsx
> 3. SearchPage.jsx
> 4. about.js
> 
> Let's hit them all with this PR. Can you also add `<SEO title="Home" />` or whatever the page is named just above `<Header />`, so that we get a title on all pages too? Then you can remove that from Layout.js.

Done in https://github.com/Seneca-CDOT/telescope/commit/ac1dd7a26f9f0ab96f9e2d71df32ae45b5c27a92

**Pull request merged**